source/support/zendesk/index.html.md.erb

```diff
@@ -1,8 +1,246 @@
+1. Submit the ticket as solved.
+
+## Macros to answer common questions quickly
+
+We’ve set up a few macros to help you answer common questions more quickly.
+
+You can adjust the wording so your response ties in to the question the user is asking.
+
+### GOV.UK Account: Account confusion/signing in
+
+This macro explains what the account is for.
+
+Use this macro when people get in touch because they think they have an account when they do not actually have one.
+
+Usually these people are confusing an existing account (for example, gateway or settled status) with a GOV.UK account.
+
+### GOV.UK Account: Not getting security code
+
+This macro explains what the user can do if they’re not getting the security code.
+
+### GOV.UK Account: More information needed
+
+This macro asks the user to provide more information.
+
+Use this macro if the initial ticket doesn’t have enough information for you to be able to understand the problem.
+
+### GOV.UK Account: Additional feedback requested
+
+This macro thanks the user for giving feedback and asks them a few more questions about their experience with the account.
+
+Use this macro if the user has given a bit of feedback about their experience.
+
+### GOV.UK Account: Closing a pending ticket
+
+This macro explains that we’re going to close the ticket but that the user can reopen it anytime if they need to.
+
+Use this macro to close __pending__ tickets that have not been updated in 5 working days or more.
+
+### GOV.UK Account: Changing details (email or phone)
+
+This macro explains what the user needs to do to update their account.
+
+Use this macro when dealing with tickets where a user asks us to update their details.
+
+### GOV.UK Account: Thanking user for additional feedback
+
+This macro thanks users for providing answers to the follow-up feedback questions.
+
+Use this macro to respond to tickets where users have answered out feedback questions.
+
+## Prepared responses to common issues
+
+### Security question
+
+Thanks for your question. We take security very seriously. We make sure any information a user submits is held securely.
+
+We have put in place security measures that defend against the kinds of threats that GOV.UK accounts could face. For example:
+
+- all data is encrypted between us and a user’s web browser using HTTPS
+- a secondary security code is required to log into an account
+- accounts lock out after a set number of failed login attempts
+- we store passwords securely
+
+You can review or report any suspicious activity in the security section of your account.
+
+You also can read about how GOV.UK accounts use cookies and personal information in the full [GOV.UK accounts privacy notice](https://www.gov.uk/government/publications/govuk-accounts-trial-full-privacy-notice-and-accessibility-statement?_ga=2.185125289.1003537497.1608298867-1303934202.1608298867).
```

Comment:
oops we might want to take the GA param of this link

Reply:
```suggestion
You also can read about how GOV.UK accounts use cookies and personal information in the full [GOV.UK accounts privacy notice](https://www.gov.uk/government/publications/govuk-accounts-trial-full-privacy-notice-and-accessibility-statement).
```

Reply:
will do